geomesa-core/src/main/scala/org/locationtech/geomesa/core/core.scala

```diff
@@ -38,17 +38,18 @@ package object core {
   val DEFAULT_CACHE_TABLE_NAME               = "geomesa.index.cache-table"
   val DEFAULT_AGGREGATOR_CLASS_PROPERTY_NAME = "geomesa.iterators.aggregator-class"
 
-  val GEOMESA_ITERATORS_SIMPLE_FEATURE_TYPE = "geomesa.iterators.aggregator-types"
-  val GEOMESA_ITERATORS_SFT_NAME            = "geomesa.iterators.sft-name"
-  val GEOMESA_ITERATORS_SFT_INDEX_VALUE     = "geomesa.iterators.sft.index-value-schema"
-  val GEOMESA_ITERATORS_ATTRIBUTE_NAME      = "geomesa.iterators.attribute.name"
-  val GEOMESA_ITERATORS_ATTRIBUTE_COVERED   = "geomesa.iterators.attribute.covered"
-  val GEOMESA_ITERATORS_ECQL_FILTER         = "geomesa.iterators.ecql-filter"
-  val GEOMESA_ITERATORS_TRANSFORM           = "geomesa.iterators.transform"
-  val GEOMESA_ITERATORS_TRANSFORM_SCHEMA    = "geomesa.iterators.transform.schema"
-  val GEOMESA_ITERATORS_IS_DENSITY_TYPE     = "geomesa.iterators.is-density-type"
-  val GEOMESA_ITERATORS_INDEX_SCHEMA        = "geomesa.iterators.index.schema"
-  val GEOMESA_ITERATORS_VERSION             = "geomesa.iterators.version"
+  val GEOMESA_ITERATORS_SIMPLE_FEATURE_TYPE      = "geomesa.iterators.aggregator-types"
+  val GEOMESA_ITERATORS_SFT_NAME                 = "geomesa.iterators.sft-name"
+  val GEOMESA_ITERATORS_SFT_INDEX_VALUE          = "geomesa.iterators.sft.index-value-schema"
+  val GEOMESA_ITERATORS_ATTRIBUTE_NAME           = "geomesa.iterators.attribute.name"
+  val GEOMESA_ITERATORS_ATTRIBUTE_COVERED        = "geomesa.iterators.attribute.covered"
+  val GEOMESA_ITERATORS_ECQL_FILTER              = "geomesa.iterators.ecql-filter"
+  val GEOMESA_ITERATORS_TRANSFORM                = "geomesa.iterators.transform"
+  val GEOMESA_ITERATORS_TRANSFORM_SCHEMA         = "geomesa.iterators.transform.schema"
+  val GEOMESA_ITERATORS_IS_DENSITY_TYPE          = "geomesa.iterators.is-density-type"
+  val GEOMESA_ITERATORS_IS_TEMPORAL_DENSITY_TYPE = "geomesa.iterators.is-temporal-density-type"
+  val GEOMESA_ITERATORS_INDEX_SCHEMA             = "geomesa.iterators.index.schema"
+  val GEOMESA_ITERATORS_VERSION                  = "geomesa.iterators.version"
```

Comment:
Yea -- I added GEOMESA_ITERATORS_IS_TEMPORAL_DENSITY_TYPE and then had to add whitespace to the rest of the entries to line everything up on the equals sign.